HW_3/task_3.py

```diff
@@ -0,0 +1,14 @@
+"""
+Задайте список из вещественных чисел, округленных до сотых.
+Найдите разницу между максимальным и минимальным значением дробной части элементов.
+
+Ввод: значение типа <list> (либо значения типа <int> – размерность списка)
+Вывод: значение типа <float>
+
+Пример:
+[1.1, 1.2, 3.1, 5, 10.01]
+2.0
+"""
+lst = list(map(float, input("Введите числа через пробел: ").split()))
+new_lst = [round(i % 1,2) for i in lst if i % 1 != 0]
```

Comment:
А почему нулевой дробной части не может быть? На семинаре я говорил, что это не является исключением и пример из условия это подтверждает.
```suggestion
new_lst = [round(i % 1,2) for i in lst]
```